pagopa/pn-frontend-e2e
# PN 9399 e 2 e destinatario visualizzazione elenco deleghe

## Short description

refactor feature in file 002_045_visualizzaElencoDeleghe

## List of changes proposed in this pull request

changed method for creating the delegation (PG to PF)
updated time for visibility of element

## How to test

click on "deleghe" in the PN's sidebar, add a delegation and verify the correct display of the delegation in the page, then revoke it

- [ x ] Does this PR require a manual test in AWS CodeBuild?
